src/Security/TokenChecker.php

```diff
@@ -25,55 +26,72 @@ class TokenChecker
     private $session;
 
     /**
-     * @param SessionInterface $session
+     * @var AuthenticationTrustResolverInterface
      */
-    public function __construct(SessionInterface $session)
+    private $trustResolveer;
+
+    /**
+     * @param SessionInterface                     $session
+     * @param AuthenticationTrustResolverInterface $trustResolveer
+     */
+    public function __construct(SessionInterface $session, AuthenticationTrustResolverInterface $trustResolveer)
     {
         $this->session = $session;
+        $this->trustResolveer = $trustResolveer;
```

Comment:
I think you mean `$trustResolver`?

Reply:
Good catch, fixed in be0c6d4de2cafccf7bef134d31f25e8c20f4f0ae